### ISSUE TITLE
Map more datasets

### ISSUE THREAD

**Issue by blotero** [blotero/seg_tgce#26]
Add API's for mapping more datasets

**Comment by blotero** [comment]
Add MORPHO MNIST support https://github.com/dccastro/Morpho-MNIST

**Comment by blotero** [comment]
Add Longitudinal Multiple Sclerosis Lesion Segmentation Challenge support: https://smart-stats-tools.org/lesion-challenge-2015

**Comment by blotero** [comment]
Add BRATS support: 


www.braintumorsegmentation.org

challenge.kitware.com/midas/folder/102

www.virtualskeleton.ch/

**Comment by blotero** [comment]
Add  LIDC-IDRI (lung abnormalities) support:



https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3041807/

**Comment by blotero** [comment]
Add QSMSC at UCL support:

https://www.ucl.ac.uk/ion/research/research-centres/queen-square-multiple-sclerosis-centre/ms-clinical-trials/queen-square-ms

**Comment by blotero** [comment]
> Add LIDC-IDRI (lung abnormalities) support:
> 
> https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3041807/
> 
> https://wiki.nci.nih.gov/display/CIP/LIDC
> 
> https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=1966254

Apparently, a tool is required for downloading the whole 125GB of images: https://wiki.cancerimagingarchive.net/display/NBIA/Downloading+TCIA+Images

**Comment by blotero** [comment]
I think we could use the tf native MNIST dataset instead of Morpho. This way, we can apply a similar disturbance strategy to the one applied to oxford pet instead of applying morphological transformations. https://www.tensorflow.org/api_docs/python/tf/keras/datasets/mnist

**Comment by blotero** [comment]
> Add support for tomato seeds: https://gcpds-image-segmentation.readthedocs.io/en/latest/notebooks/02-datasets.html#Tomato-Seeds

This dataset is way too small for being meaningful, a typical UNet actchitecture training will have loss curves like the following:


![image](https://github.com/blotero/seg_tgce/assets/43280129/c2305acd-aced-48b2-9c31-00cf047e68aa)



This is unacceptable for achieving decent annotators emulation. 


We will be using the agumented version with detection target instead. However, it would still require to adapt it as tf generator dataset.

**Comment by blotero** [comment]
> > Add support for tomato seeds: https://gcpds-image-segmentation.readthedocs.io/en/latest/notebooks/02-datasets.html#Tomato-Seeds
> 
> This dataset is way too small for being meaningful, a typical UNet actchitecture training will have loss curves like the following:
> 
> ![image](https://private-user-images.githubusercontent.com/43280129/302575206-c2305acd-aced-48b2-9c31-00cf047e68aa.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcyMzUzMTgsIm5iZiI6MTcwNzIzNTAxOCwicGF0aCI6Ii80MzI4MDEyOS8zMDI1NzUyMDYtYzIzMDVhY2QtYWNlZC00OGIyLTljMzEtMDBjZjA0N2U2OGFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA2VDE1NTY1OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE4ZTQwODQ5MGU1MWRiYjg0Y2RiY2YzMTY0OTc5YzM4NjRmOTI0ZDI3YWE5ZjJkMTg3MmUyYzAwOTgyOTAyZGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.VUR_7oqWaqEFQVVmZwUvuQ6RgcHozLTM_SPkCKBe4oE)
> 
> This is unacceptable for achieving decent annotators emulation.
> 
> We will be using the augmented version with detection target instead. However, it would still require to adapt it as tf generator dataset.

Successful results for tomato seeds disturbing model:


![image](https://github.com/blotero/seg_tgce/assets/43280129/4fe2beef-a8ff-4cc3-ad25-7bfec3ede001)


Implementation will be included in nexts PR's

**Comment by blotero** [comment]
Map Crodw Seg dataset: 

https://github.com/wizmik12/CRowd_Seg/tree/main 

https://drive.google.com/drive/folders/17VukoKpwZclRrDcWSK1aYd_lPeqWNM8N?usp=sharing=
